tests/testutil/service.go

```diff
@@ -467,24 +479,25 @@ func (ts *TestService) GetKubeAppEndpointURLs() []string {
 	return strings.Split(urls[2:len(urls)-2], " ")
 }
 
+// todo: needs to get updated so it takes into account all service weights
 // GetKubeCurrentWeight does the exact same thing as GetCurrentWeight but uses the kubectl command instead of rio
-func (ts *TestService) GetKubeCurrentWeight() int {
-	ts.reload()
-	args := []string{"get", "services.rio.cattle.io", ts.Service.Name, "-n", TestingNamespace, "-o", "json"}
-	resultString, err := KubectlCmd(args)
-	if err != nil {
-		ts.T.Fatalf("Failed to get services.rio.cattle.io:  %v", err.Error())
-	}
-	var svc riov1.Service
-	err = json.Unmarshal([]byte(resultString), &svc)
-	if err != nil {
-		ts.T.Fatalf("Failed to unmarshal service results: %s with error: %v", resultString, err.Error())
-	}
-	if svc.Status.ComputedWeight != nil {
-		return *svc.Status.ComputedWeight
-	}
-	return 0
-}
+//func (ts *TestService) GetKubeCurrentWeight() int {
```

Comment:
@izaac / @rancher-max I had to comment this out because computedWeight is now relative to all other services (ie: 2 svcs at 100 equals 50% each). This weight percentage is calculated by rio CLI which is what `GetCurrentWeight` now uses. Do we want to do this calc with kubectl too or should we remove ?

Reply:
It's ok to comment for now we would need to check if there's a way to get this same value with kubectl, manual testing involved, and validate that against the new CLI calculation.